dockers/docker-dhcp-relay/docker-dhcp-relay.supervisord.conf.j2

```diff
@@ -38,31 +38,73 @@ stderr_logfile=syslog
 dependent_startup=true
 dependent_startup_wait_for=rsyslogd:running
 
+{% set num_relays = { 'count': 0 } %}
 {# If our configuration has VLANs... #}
 {% if VLAN %}
 {# Count how many VLANs require a DHCP relay agent... #}
-{% set num_relays = { 'count': 0 } %}
 {% for vlan_name in VLAN %}
 {% if VLAN[vlan_name]['dhcp_servers'] %}
 {% set _dummy = num_relays.update({'count': num_relays.count + 1}) %}
 {% endif %}
 {% endfor %}
-{# If one or more of the VLANs require a DHCP relay agent... #}
+{% endif %}
+{# If our configuration has Portchannels... #}
+{% if PORTCHANNEL_INTERFACE %}
```

Comment:
can you please split the configuration into three files one per each port class/type; vlan, physical, pc.

Reply:
Can we have multiple supervisord conf files for a docker? Please provide some pointers for it.

Reply:
the idea is that all `*.conf` files under `/etc/supervisor/conf.d/` are included be this [line](https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-base-buster/etc/supervisor/supervisord.conf#L29). So, I'd suggest the following:

1. Create two more new *.conf; one for physical interface and another for port channels.
2. Each should be self-contained; so we would create new process groups in each file
3. there is no need to include the syslog section that is in the current conf file

Reply:
Done the changes as suggested.